src/libraries/System.Private.CoreLib/src/System/ReadOnlySpan.cs

```diff
@@ -23,7 +23,7 @@ namespace System
     public readonly ref struct ReadOnlySpan<T>
     {
         /// <summary>A byref or a native ptr.</summary>
-        internal readonly ByReference<T> _reference;
+        internal readonly ref T _reference;
```

Comment:
Should this be `readonly ref readonly`?

Reply:
This can't be done. There are several places in this file that fail with that. The indexer for example.

```csharp
return ref Unsafe.Add(ref _reference, (nint)(uint)index /* force zero-extension */);
```

Reply:
Deferring to future.